web/ui/react-app/src/pages/rules/RulesContent.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { FC } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { APIResponse } from '../../hooks/useFetch';
+import { Alert, Table, Badge } from 'reactstrap';
+import { Link } from '@reach/router';
+import { formatRelative, createExpressionLink, humanizeDuration } from '../../utils';
+import { Rule } from '../../types/types';
+import { now } from 'moment';
+
+interface RulesContentProps {
+  response: APIResponse<RulesMap>;
+}
+
+interface RuleGroup {
+  name: string;
+  file: string;
+  rules: Rule[];
+  evaluationTime: string;
+  lastEvaluation: string;
+}
+
+export interface RulesMap {
+  groups: RuleGroup[];
+}
+
+const GraphExpressionLink: FC<{ expr: string; title: string }> = props => {
```

Comment:
This component should look like:
```
export const GraphExpressionLink: FC<{ expr: string; title: string; name: string  & PathPrefixProps}> = memo(({ name, title, expr, pathPrefix }) => {
  return (
    <div>
      {title}:
      <Link className="ml-4" to={createExpressionLink(expr, pathPrefix)}>
        {name}
      </Link>
    </div>
  );
});
```

```name``` and ```expr``` may have different content, so we can't rely on ```expr``` to represent the ```name```
Also, wrapping the ```Link``` and ```title``` in div will remove the need of ```<br/>```. IMO it's better.
Comparing with the old UI titles here are wrapped in ```<strong>```, maybe you should remove the extra tag

Another, more important thing is to pas the ```pathPrefix``` prop down to ```createExpressionLink```. IMO the ``../``` in ```createExpressionLink``` should be passed from outside and the function just to return
``` ${prefix}graph?g0.expr=${encodeURIComponent(expr)}&g0.tab=1&g0.stacked=0&g0.range_input=1h ```
The usage should looks like ```createExpressionLink(expr, pathPrefix + '/../')```